.circleci/config.yml

```diff
@@ -153,6 +156,7 @@ jobs:
           no_output_timeout: 40m
       - run:
           name: Running integration tests - <<parameters.test_tags>>
+          namespace: "nsm-system-integration"
```

Comment:
i'm sorry I probably was not clear. This is not needed here. It is needed in the workflow (at the bottom of the ffile) where you invoke `integration-tests` for the different scenarios and cloud providers. As we agreed all `integration-tests` for packet will have "nsm-system-integration" while the rest will not be set and will default to "default".